turbo/generators/templates/README.md.hbs

```diff
@@ -13,14 +13,14 @@
     <a href="https://www.npmjs.com/package/@microsoft/teams.{{name}}" target="_blank">
         <img src="https://img.shields.io/npm/dw/@microsoft/teams.{{name}}" />
     </a>
-    <a href="https://microsoft.github.io/teams-ai" target="_blank">
+    <a href="https://microsoft.github.io/teams.ts" target="_blank">
         <img src="https://img.shields.io/badge/📖 docs-open-blue" />
     </a>
 </p>
 
 {{description}}
 
-<a href="https://microsoft.github.io/teams-ai" target="_blank">
+<a href="https://microsoft.github.io/teams.ts/2.getting-started/1.create-application.html" target="_blank">
```

Comment:
this will be quickstart after my changes get added in
```suggestion
<a href="https://microsoft.github.io/teams.ts/2.getting-started/1.quickstart.html" target="_blank">
```